card/spec/lib/card/subcards_spec.rb

```diff
@@ -56,6 +56,29 @@
       end
       expect(Card['supermen+pseudonym'].name).to eq 'supermen+pseudonym'
     end
+
+    it "cleans the cache for autonaming case" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.